README.md

```diff
@@ -335,6 +335,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 
 [Communication software](https://en.wikipedia.org/wiki/Communication_software) used to provide remote access to systems and exchange files and messages in text, audio and/or video formats between different computers or users, using their own custom protocols.
 
+- [ADPS](https://adps-project.org/) - Amateur Digital Post Service is an offline communication software which has 2 separate implementations, one is written on C# (GUI) and another is written on Python (CLI). ([GUI Client Source Code](https://github.com/ivanmihval/SharpADPS), [CLI Client Source Code](https://github.com/ivanmihval/PyADPS)) `MIT` `C#/Python`
```

Comment:
- The description should explain what the software does (maybe summarize https://adps-project.org/ in one sentence)
- There is no documentation at all for the C# repository, all projects should have at the very least working installation instructions, preferably full instalaltion/usage/contributing/... documentation (have a look at [this template](https://github.com/nodiscc/toolbox/blob/master/TEMPLATES/DEV/README.md))
- Only one `[Source Code]` link is allowed, so I think you would have to split this to 2 separate entries
- Lastly, while the concept is interesting and addresses real-world problems, I think it falls outside the scope of this list:

> Instead of TCP packets on traditional networks like the internet you write your messages to your USB-flash and exchange with other participants.

> This is a list of [Free](https://en.wikipedia.org/wiki/Free_software) Software [network services](https://en.wikipedia.org/wiki/Network_service) and [web applications](https://en.wikipedia.org/wiki/Web_application) which can be hosted on your own server(s).

The definition of "network service" implies that client/server machines are somehow interconnected, so a sneakernet solution, while interesting, would not fall under this definition. I think you should post it on the communities linked from https://github.com/awesome-selfhosted/awesome-selfhosted#external-links becauses it deserves some interest - but I don't think we should merge it

Reply:
I agree with your points, as this is the proper way to go. External links might be the best way to use this instead of a traditional way of web applications. I thank you for your insight into this, nodiscc.

Reply:
Hi @nodiscc! Thank you for your feedback.

> There is no documentation at all for the C# repository, all projects should have at the very least working installation instructions, preferably full instalaltion/usage/contributing/... documentation (have a look at [this template](https://github.com/nodiscc/toolbox/blob/master/TEMPLATES/DEV/README.md))

You're right that any project should have a README for people who has never the project before. So I've created this file, I've also recorded a demo and put to README.

> I think you should post it on the communities linked from https://github.com/awesome-selfhosted/awesome-selfhosted#external-links becauses it deserves some interest - but I don't think we should merge it

Yep, this week I knew about the [Alternative Internet List](https://github.com/redecentralize/alternative-internet) which exists in the external links of this repository. So the last commit of this repo is mine.

> The definition of "network service" implies that client/server machines are somehow interconnected, so a sneakernet solution, while interesting, would not fall under this definition.

I understand your point, what if I create a web service which works with this network, can I create a MR with something like "Internet gate for ADPS sneakernet network"?

Reply:
> what if I create a web service which works with this network, can I create a MR

Yes.